src/status_im/wallet/accounts/core.cljs

```diff
@@ -19,7 +19,17 @@
             [status-im.multiaccounts.recover.core :as recover]
             [status-im.ethereum.mnemonic :as mnemonic]
             [taoensso.timbre :as log]
-            [status-im.wallet.prices :as prices]))
+            [status-im.wallet.prices :as prices]
+            [status-im.utils.hex :as hex]
+            [status-im.ethereum.ens :as ens]
+            [status-im.ethereum.resolver :as resolver]))
+
+(re-frame/reg-fx
```

Comment:
We have a similar to this one, an effect in `status-im.ens.core` and other places too, I think for future easier maintenance maybe it would make sense to use it only from `ens` namespaces which will get the registry and parse name.

Reply:
fixed